awesomplete.js

```diff
@@ -109,6 +109,14 @@ var _ = self.Awesomplete = function (input, o) {
 		inside: this.container
 	});
 	
+	this.status = $.create("span", {
+		hiddena11y: "",
```

Comment:
Why? This element doesn’t get focus either. Why can’t we apply whatever this element does to the list item itself? I.e. make each list item an aria-live element when it's selected and remove those attributes when its deselected.